src/main/zapHomeFiles/hud/tools/toggleScript.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * HTML Report tool
+ *
+ * When selected displays the standard HTML report in a new window / tab
+ */
+
+const ToggleScript = (function () {
+	// Constants
+	const NAME = 'toggleScript';
+	const LABEL = 'Toggle Script';
```

Comment:
This should be internationalised

Reply:
What do you mean by internationalised? Should I add the messages to the UIMessages.properties file so that translations can be easily added?

Reply:
Yes, you can then get string with `I18n.t('my_key')`.

Reply:
Gotcha, thanks!